android/src/main/kotlin/com/google/flutter/recaptcha/RecaptchaEnterprisePlugin.kt

```diff
@@ -59,18 +59,26 @@ class RecaptchaEnterprisePlugin : FlutterPlugin, MethodCallHandler, ActivityAwar
     }
     val siteKey = call.argument<String>("siteKey")
     val timeout = call.argument<Double>("timeout")
```

Comment:
Remember that the fetchClient doesn't require a timeout.

Reply:
This was being ignored in the code, but now changing to have one method for each API.